docker/Dockerfile.builder

```diff
@@ -1,7 +1,7 @@
-ARG NODE_BASE=10.15.3-alpine
+ARG NODE_BASE=10-slim
 FROM node:${NODE_BASE}
 
 # Use --no-cache to avoid rm -rf /var/cache/apk/*
-RUN apk --update --no-cache add python make g++
+RUN apk --update --no-cache install python make g++
```

Comment:
It's more like `apt-get install  python make g++` than `apk add ...`. Please change `apk add` in `.drone.yml` as well.